rasa/core/policies/rule_policy.py

```diff
@@ -739,12 +741,14 @@ def _find_action_from_loop_happy_path(
 
     def _find_action_from_rules(
         self, tracker: DialogueStateTracker, domain: Domain
-    ) -> Tuple[Optional[Text], Optional[Text]]:
+    ) -> Tuple[Optional[Text], Optional[Text], bool]:
         tracker_as_states = self.featurizer.prediction_states([tracker], domain)
         states = tracker_as_states[0]
 
         logger.debug(f"Current tracker state: {states}")
 
+        entered_unhappy_path = False
```

Comment:
please add comment why it is here
```suggestion
        returning_from_unhappy_path = False
```